tests/apps/perf/tester/Dockerfile

```diff
@@ -10,11 +10,11 @@ FROM golang:1.15 as fortio_build_env
 WORKDIR /fortio
 ADD "https://api.github.com/repos/fortio/fortio/branches/master" skipcache
```

Comment:
What is it for? Add string to file?

Reply:
To force a cache miss so the code below is always executed in local development.